python/ray/serve/deployment.py

```diff
@@ -416,8 +423,31 @@ def options(
                 "into `serve.run` instead."
             )
 
-        if num_replicas not in [DEFAULT.VALUE, None]:
+        # Modify max_concurrent_queries and autoscaling_config if
+        # `num_replicas="auto"`
+        if num_replicas == "auto":
```

Comment:
can you pull this into a helper method and document the behavior?

Reply:
Done! Pulled to `utils.py`.